include/genomics/printer.hpp

```diff
@@ -92,67 +95,97 @@ namespace genomics {
       return -(delim + 1);
     }
 
-    std::string off_target_string(genome_structure gs,
-                                  std::vector<std::vector<std::tuple<int64_t, match>>>& off_targets,
-                                  int64_t max_off_targets) {
+    float calculate_cfd(std::string sgRNA, std::string sequence, std::string PAM) {
```

Comment:
I think a better place for this function is in `process.hpp`, but I'm having circular import issues when I try that. Perhaps we can open a new issue if this is a concern.